spec/features/authenticated_user/auth_non_admin_views_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature "the authenticated user" do
+  let(:valid_attributes) {
+    { full_name: "Stan The Man",
+      user_name: "Theman",
+      password: "password",
+      role: 0,
+      email: "stan@stan.com" }
+  }
+
+  scenario "stays on page when attempting to access any /admin route" do
+    User.create(valid_attributes)
+
+    visit items_path
+    click_link "Login"
+    fill_in "Username", with: "Theman"
+    fill_in "Password", with: "password"
+    click_button "Login"
+    visit '/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.